### PR TITLE
Irish is love, Irish is life.

### PR DESCRIPTION
**Pull request by callumok2004**

### CODE REVIEW

File: websites/P/PornHub/dist/metadata.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "fr": "PornHub est le site numéro un de pornographie dans le monde avec des centaines de milliers de vidéos gratuites, incluant l'option HD gratuite et la possibilité de télécharger les vidéos.",
     "uk_UA": "PornHub - це безкоштовний порно-сайт, номер один у світі, який містить сотні тисяч безкоштовних відео, включаючи безкоштовну HD якість та можливості завантаження.",
     "nl": "PornHub is de nummer één gratis pornosite ter wereld met honderdduizenden gratis video's, inclusief gratis HD en downloadmogelijkheden.",
-    "zh_CN": "PornHub是世界上第一的免费色情网站，拥有成千上万的免费色情视频，甚至还包括免费高清观看和下载的功能。"
+    "zh_CN": "PornHub是世界上第一的免费色情网站，拥有成千上万的免费色情视频，甚至还包括免费高清观看和下载的功能。",
+    "ga_IE": "Is é PornHub an láithreán porn saor in aisce is mó ar domhan ina bhfuil na céadta mílte físeán saor in aisce, lena n-áirítear HD saor in aisce agus cumais íoslódála."
   },
   "url": [
     "www.pornhub.com",
```
Comment (Bas950) [inline websites/P/PornHub/dist/metadata.json:19]: inc version

**Review by ririxidev — CHANGES_REQUESTED** [review]
@callumok2004,
You should check out thing(s) that I requested to change in my review. Please look at them!
 - Squash all commits. (https://medium.com/@mittalyashu/how-to-squash-commits-in-a-github-pull-request-97b36576eacb)

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by callumok2004** [comment]
> @callumok2004,
> You should check out thing(s) that I requested to change in my review. Please look at them!
> 
> * Squash all commits. (https://medium.com/@mittalyashu/how-to-squash-commits-in-a-github-pull-request-97b36576eacb)
> 
> Thanks,
> **ririxi**. PreMiD Reviewer Team.
> Contact: [mail@ririxi.dev](mailto:mail@ririxi.dev), ririxi#2721

thanks this officially broke my git :(

not gonna be squashed, deal wiv it

**Comment by callumok2004** [comment]
nevermind there done

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by StrikerFRFX — APPROVED** [review]
lgtm

**Pull request merged**